opengrok-web/src/main/java/org/opengrok/web/api/v1/RestApp.java

```diff
@@ -34,6 +36,7 @@
     
     public RestApp() {
         register(new SuggesterAppBinder());
+        register(new InstrumentedResourceMethodApplicationListener(Metrics.getInstance()));
```

Comment:
in short, what does it do? `InstrumentedResourceMethodApplicationListener`

Reply:
After the changes for Micrometer this file remains unchanged (remove the copyright change).